### PR TITLE
pages/faq: clarify stance on upgrading directly from Atomic Host 

### PR DESCRIPTION
**Pull request by dustymabe**
Fixes https://github.com/coreos/fedora-coreos-tracker/issues/257

### CODE REVIEW

**Comment by dustymabe** [comment]
cc @keithy

File: modules/ROOT/pages/faq.adoc [hunk 1]
```diff
@@ -130,9 +130,11 @@ https://discussion.fedoraproject.org/t/launch-faq-how-do-i-migrate-from-containe
...
 to the cloud-init/Ignition transition at least. Since Fedora CoreOS will
 be using rpm-ostree technology, it may be possible to rebase from Fedora
-Atomic Host to Fedora CoreOS. This will be part of a "migrating from
-Fedora Atomic Host" guide which will be published as soon as an actual
-version of Fedora CoreOS is available.
+Atomic Host to Fedora CoreOS, but it will not be recommended. It will
+be preferable to gain experience deploying systems using Ignition so
+that they can be re-provisioned easily if needed. This will all be part
+of a "migrating from Fedora Atomic Host" guide which will be published as
+soon as an actual version of Fedora CoreOS is available.
```
Comment (bgilbert) [inline modules/ROOT/pages/faq.adoc]: An actual FCOS version is available, so this is no longer accurate.

Comment (dustymabe) [reply modules/ROOT/pages/faq.adoc]: ok modified it to be less specific

Comment (dustymabe) [reply modules/ROOT/pages/faq.adoc]: ok modified it to be less specific

**Comment by keithy** [comment]
The reason that I raised this issue was the process of deciding whether to wait and rebase my exisitng Fedora Atomic server, or take the plunge with a new install. Sincethe above text still says "may be", I am none the wiser.

**Comment by bgilbert** [comment]
@keithy There isn't a really great answer right now.  We don't know whether rebasing will be feasible, but it will certainly not be the preferred approach.  On the other hand, Fedora CoreOS is in preview and shouldn't be used for production workloads.

**Pull request merged**